### PR TITLE
Add support for `JSON.MERGE`

### LINKED ISSUE #2500: Add support for JSON.MERGE
As defined in https://github.com/RedisJSON/RedisJSON/blob/e4608065002bdf585ad54fbe142e08be18d666a8/commands.json#LL169C1-L169C1

### PR DESCRIPTION
**Pull request by shacharPash**
### Description

> Describe your pull request here

---

### Checklist

- [ ] Does `npm test` pass with this change (including linting)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

Closes #2500

### CODE REVIEW

File: packages/json/lib/commands/MERGE.spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        );
+    });
+
+    testUtils.testWithClient('client.json.merge', async client => {
+        assert.equal(
+            await client.json.merge('key', '$', 'json'),
+            'OK'
+        );
+    }, GLOBAL.SERVERS.OPEN);
+});
```
Comment (leibale) [inline packages/json/lib/commands/MERGE.spec.ts:21]: empty line please.. :)

File: packages/json/lib/commands/MERGE.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { RedisJSON, transformRedisJsonArgument } from '.';
+
+export const FIRST_KEY_INDEX = 1;
+
+export function transformArguments(key: string, path: string, json: RedisJSON): Array<string> {
+    return ['JSON.MERGE', key, path, transformRedisJsonArgument(json)];
+}
+
+export declare function transformReply(): 'OK' | null;
```
Comment (leibale) [inline packages/json/lib/commands/MERGE.ts:9]: `null`?! + empty line..

**Review by leibale — CHANGES_REQUESTED** [review]

**Review by leibale — APPROVED** [review]

**Comment by kiyaGu** [comment]
Hi @shacharPash @leibale, do you know when this feature is going to be available?

**Comment by leibale** [comment]
@kiyaGu we're waiting for RedisJSON 2.6 to be released, see [here](https://github.com/RedisJSON/RedisJSON/releases)

**Comment by arsengoian** [comment]
@leibale looks like it has been released several weeks ago

**Comment by znycheporuk** [comment]
Any updates?

> @kiyaGu we are waiting for RedisJSON 2.6 to be released, see [here](https://github.com/RedisJSON/RedisJSON/releases)

**Pull request merged**